src/schematools/types.py

```diff
@@ -311,7 +311,6 @@ def build_through_table(
         from schematools.utils import get_rel_table_identifier, to_snake_case, toCamelCase
 
         def _get_fk_target_table(dataset_id: str, table_id: str) -> DatasetTableSchema:
-            """Gets the"""
```

Comment:
I was once given a largeish codebase to maintain after its author had left the company. Apparently, they had turned some autocomment feature on in their IDE, and blindly committed everything it produced, so every method had an docstring like

```python
def get_table_from_file(table, filename, ignore_errors):
    """Gets the table.

    Args:
        table: The table.
        filename: The name.
        ignore_errors: The ignore errors.
    """
```

There was also a method called `get`, with this exact docstring :)